cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -26,19 +26,584 @@ Cask Data Application Platform Release Notes
 ------------
 
+- `CDAP-3361 <https://issues.cask.co/browse/CDAP-3361>`__ -
+  CDAP-UI should use 'name' instead of 'id' fields when listing app programs.
+  
+- `CDAP-3396 <https://issues.cask.co/browse/CDAP-3396>`__ -
+  CDAP-UI scroll bar for logs has jitters and is unbounded at the top.
+  
+- `CDAP-3409 <https://issues.cask.co/browse/CDAP-3409>`__ -
+  java.lang.NoClassDefFoundError: co/cask/cdap/data/dataset/DatasetCreationSpec
+  
+- `CDAP-3492 <https://issues.cask.co/browse/CDAP-3492>`__ -
+  Executing 'select \*' from a dataset with many fields generates an error.
+  
+- `CDAP-3499 <https://issues.cask.co/browse/CDAP-3499>`__ -
+  TestFrameworkTestRun testGetServiceURL fails sporadically.
+  
+- `CDAP-3500 <https://issues.cask.co/browse/CDAP-3500>`__ -
+  TestFrameworkTestRun testWorkerStop fails sporadically on Bamboo builds.
+  
+- `CDAP-3534 <https://issues.cask.co/browse/CDAP-3534>`__ -
+  CDAP-UI metric shown in output graph in flowlets is not the correct one.
+  
+- `CDAP-3573 <https://issues.cask.co/browse/CDAP-3573>`__ -
+  Worker RuntimeException behavior in Standalone and Distributed are not the same.
+  
+- `CDAP-3578 <https://issues.cask.co/browse/CDAP-3578>`__ -
+  CDAP-UI Service explore dialog "view" icon is broken after download is clicked.
+  
+- `CDAP-3601 <https://issues.cask.co/browse/CDAP-3601>`__ -
+  Large number of Javadoc errors in modules
+  
+- `CDAP-3602 <https://issues.cask.co/browse/CDAP-3602>`__ -
+  The Documentation Search Engine does not find all pages with the search term.
+  
+- `CDAP-3633 <https://issues.cask.co/browse/CDAP-3633>`__ -
+  CDAP-SDK complains about node version 4.0.0 which is greater that 0.10.
+  
+- `CDAP-3638 <https://issues.cask.co/browse/CDAP-3638>`__ -
+  HttpExceptionHandler does not handle FORBIDDEN exceptions.
+  
+- `CDAP-3640 <https://issues.cask.co/browse/CDAP-3640>`__ -
+  Standalone cdap.sh doesn't allocate enough memory.
+  
+- `CDAP-3641 <https://issues.cask.co/browse/CDAP-3641>`__ -
+  WorkflowStatsSLAHttpHandler hangs if units not provided.
+  
+- `CDAP-3642 <https://issues.cask.co/browse/CDAP-3642>`__ -
+  After stopping a flow, the CDAP-UI shows status as 'Killed'.
+  
+- `CDAP-3656 <https://issues.cask.co/browse/CDAP-3656>`__ -
+  The ProgramManager does not get notified when a worker stops on its own.
+  
+- `CDAP-3262 <https://issues.cask.co/browse/CDAP-3262>`__ -
+  There is a problem under Microsoft Windows and using the CDAP Standalone scripts when  
+  JAVA_HOME is defined as a path with spaces in it. A workaround is to use a definition of 
+  JAVA_HOME that does not include spaces.
+
+- `CDAP-3665 <https://issues.cask.co/browse/CDAP-3665>`__ -
+  When a MapReduce job fails, CDAP says all mappers/reducers were successful.
+  
+- `CDAP-3689 <https://issues.cask.co/browse/CDAP-3689>`__ -
+  Metadata entity types should be lower-case, not upper-case.
+  
+- `CDAP-3697 <https://issues.cask.co/browse/CDAP-3697>`__ -
+  CDAP Explore is broken on secure CDH 5.1.
```

Comment:
It would probably be in this section:
http://docs.cask.co/cdap/3.2.0-SNAPSHOT/en/admin-manual/installation/installation.html#system-requirements

the Hadoop/HBase Environment chart shows the versions of Hive that we support. Add a footnote to the chart that we don't support secure Hive 0.12.